.coveragerc

```diff
@@ -10,7 +10,7 @@ source =
   */Lib/site-packages/yarl
 
 [report]
-fail_under = 98.97
+fail_under = 98.96
```

Comment:
2024-09-03T05:38:12.7709570Z [31m[1mFAIL Required test coverage of 98.96% not reached. Total coverage: 98.96%

Well thats a bit of an unexpected error

Reply:
This fails before coverage upload so its impossible to see whats missing on python 3.8 without installing it locally

Reply:
@bdraco yes, `covdefaults` sets this to 100% but since not every job we have runs all the tests, it's impossible to have this value. I lowered it per what I saw in the CI while integrating. It's okay to lower it further when needed, since Codecov looks at the combined numbers anyway. It might be possible to pass more granular values via CLI args to pytest, but I haven't experimented with that, it's probably not worth it.